Python/omevv/v1/scripts/driftdetection.py

```diff
@@ -0,0 +1,99 @@
+        self.headers = {constants.vcGuidHeader: vCenterUUID}
+        self.headers["Authorization"] = basicAuth
+        self.headers["Content-Type"] = 'application/json'
+        self.omeIp = omeIp
+        self.uuid = vCenterUUID
+        self.payload = payload
+        self.baseline_profile_id = baseline_profile_id
+        self.monday = monday
+        self.tuesday = tuesday
+        self.wednesday = wednesday
+        self.thursday = thursday
+        self.friday = friday
+        self.saturday = saturday
+        self.sunday = sunday
+        self.time = time
+        self.client = AuthenticatedClient(base_url=base_url, token=None, verify_ssl=False). \
+            with_headers(headers=self.headers). \
+            with_timeout(constants.generalTimeOut_sec)
+
+    def create_payload(self):
+        self.payload["jobSchedule"] = {
+            "monday": self.monday,
+            "tuesday": self.tuesday,
+            "wednesday": self.wednesday,
+            "thursday": self.thursday,
+            "friday": self.friday,
+            "saturday": self.saturday,
+            "sunday": self.sunday,
+            "time": self.time
+        }
+
+    def modifyDriftDetectionSchedule(self):
+        headers = self.headers
+        url = 'https://%s/omevv/GatewayService/v1/Consoles/%s/BaselineProfiles/%s'%(self.omeIp, self.uuid, self.baseline_profile_id)
+        try:
+            response = requests.put(url, data=json.dumps(self.payload), headers=headers, verify=False);
+            status_code = response.status_code
+            if status_code == 200:
+              return "Drift detection job schedule is updated successfully"
+            elif status_code == 400 or status_code == 500 or status_code == 404:
+                data = response.json()
+                return "Error occured while modifying drift detection schedule : "+ str(data)
+            else:
+                data = response.json()
+                raise Exception("Error occured while creating manage job ",data);
+        except Exception as e:
+            print(e);
+ 
+if __name__ == "__main__":
+    PARSER = argparse.ArgumentParser(description=__doc__,
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    PARSER.add_argument("--ip", "-i", required=True, help="OME Appliance IP")
+    PARSER.add_argument("--vcusername", "-u", required=True,
+                        help="username of vcenter")
+    PARSER.add_argument("--vcpassword", "-p", required=True,
+                        help="password of vcenter")
+    PARSER.add_argument("--vcUUID", "-d", required=True, default=None,
+                        help="UUID of the relevant vCenter")
+    PARSER.add_argument("--baseline_profile_id", "-b", required=True, default=None, help="baseline profile id of baseline profile for which drift detection schedule needs to be modified")
+    PARSER.add_argument("--monday", "-mon", required=False, default=None, help="indicate the boolean value for monday")
```

Comment:
As these input variables required is false, do you need to set the default value to false?